doc/website/getting-started/overview.md

```diff
@@ -0,0 +1,540 @@
+
+```
+iox::popo::TypedPublisher<CounterTopic> publisher({"Group", "Instance", "CounterTopic"});
+publisher.offer();
+```
+
+Note that it suffices to set the first two identifiers (Group and Instance) to some default values for all topics.
+#### Sending data
+
+Now we can use the publisher to send the data in various ways.
+
+1. Loan and assign
+
+    ```
+    auto result = publisher.loan();
+    if (!result.has_error())
+    {
+        auto& sample = result.value();
+        sample->counter = 73;
+        sample.publish();
+    } else 
+    {
+        //handle the error
+    }
+    ```
+
+    Here result is an ``expected`` and hence we may get an error which we have to handle. This can happen if we try
+    to loan to many samples and exhaust memory.
+    
+    If we successfully get a sample, we can use ``operator->`` to access the underlying data and set it to the value
+    we want to send. It is important to note that in the untyped case we get a default constructed topic and such an
+    access is legal.
+    
+    Once we are done constructing and preparing the data we publish it, causing it to be delivered to any subscriber
+    which is currently subscribed to the topic.
+
+
+2. Functional approach with loaning
+
+    ```
+    publisher.loan()
+            .and_then([&](auto& sample) { 
+                auto ptr = sample.get();
+                *ptr = CounterTopic(73);
+                sample.publish();
+            })
+            .or_else([](iox::popo::AllocationError) {
+                /* handle the error */
+            });
+    ```
+
+    We try to loan a sample from the publisher and if successful get the underlying pointer ``ptr`` to our topic and
+    if successful assign it a new value. Note that ``ptr`` points to an already default constructed sample, so we
+    cannot treat it as uninitialized memory and therefore must assign the data to send.
+    
+    If you are only using a simple data type which does not rely on RAII, you can also use the pointer to construct
+    the data via emplacement new instead.
+
+    ```
+    new (ptr) CounterTopic(73);
+    ```
+
+3. Publish by copy
+
+    ```
+    CounterTopic counter(73);
+    publisher.publishCopyOf(counter);
+    ```
+    This copies the constructed ``counter`` object and hence should mostly be used for small data.
+
+4. Publish the result of a computation
+
+    ```
+    auto myComputation = [](CounterTopic *data) { *data = CounterTopic(73); };
+    auto result = publisher.publishResultOf(myComputation);
+    if(result.has_error()) {
+        //handle the error
+    }
+    ```
+
+    This can be used if we want to set the data by some callable (i.e. lambda, function or functor). As with all the
+    other ways, it can fail when there is no memory for the sample availabe and this failure must be handled.
+
+#### Creating a subscriber
+
+We now create a corresponding subscriber, usually in another application (it will work in the same application as
```

Comment:
Please either remove the braces or remove the sentence

Reply:
New sentence. The reason that I mention it is that it may be useful in some esoteric circumstance to make the code concise (i.e. you are already sending externally, then you might as well use and internal subscriber in another thread instead of shuffling the data by yourself somehow). The overhead due to the routing of the call to Roudi and back is minor (but exists).

I just wanted to avoid to give the impression that you always **have to** send to other apps.